infinitest-intellij/src/main/java/org/infinitest/intellij/idea/IdeaModuleSettings.java

```diff
@@ -51,6 +51,9 @@
 	private final Module module;
 	private final InfinitestJarLocator locator = new InfinitestJarLocator();
 
+  private static final Boolean TEST_CLASSES = true;
+  private static final Boolean MAIN_CLASSES = false;
```

Comment:
It seems that you have something wrong (inconsistent) with formatting in that part.

Btw, you can squash those two commits into one.